src/chapters/14-provider-components.md

```diff
@@ -232,7 +226,7 @@ git add app/components/rentals.hbs
 git add app/components/rentals.js
 ```
 
-## Adding the <Rentals::Filter> Component
+## Adding the <Rentals::Filter> Provider Component
 
 Now that our search query is wired up to our `<Rentals>` component, we can get into the really fun stuff! Namely, we can make our component *filter* results based on our search query. In order to encapsulate this functionality, we'll create another component called `<Rentals::Filter>`.
 
```

Comment:
Can't comment on the folded part below, but:

> Instead, we're `yield`ing to something, using the `{{yield}}` keyword...

👇 

> Instead, we're yielding to something, using the `{{yield}}` keyword

Reply:
Also:

> ...let's take a look at how we use the `{{yield}}` keyword in the `<Rentals>` component.

This sentence is a bit confusing, in that, the code snippet immediately below it doesn't actually contains a `{{yield}}`.